app-android/src/main/java/com/ustadmobile/port/android/view/SiteEnterLinkFragment.kt

```diff
@@ -93,4 +126,81 @@ class SiteEnterLinkFragment : UstadBaseFragment(), SiteEnterLinkView{
         siteLink = null
         mBinding = null
     }
-}
\ No newline at end of file
+}
+
+@Composable
+private fun SiteEnterLinkScreen(
+    uiState: SiteEnterLinkUiState = SiteEnterLinkUiState(),
+    onClickNext: () -> Unit = {},
+    onClickNewLearningEnvironment: () -> Unit = {},
+    onEditTextValueChange: (String) -> Unit = {},
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(16.dp)
+            .verticalScroll(rememberScrollState()),
+        horizontalAlignment = Alignment.CenterHorizontally
+    )  {
+
+        Image(
+            painter = painterResource(id = R.drawable.illustration_connect),
+            contentDescription = null,
+            modifier = Modifier
+                .height(200.dp))
+
+        Text(stringResource(R.string.please_enter_the_linK))
+
+        UstadTextEditField(
+            value = "",
+            label = stringResource(id = R.string.site_link),
+            onValueChange = onEditTextValueChange,
+            error = uiState.errorMessage,
+            enabled = false,
+        )
+
+        Spacer(modifier = Modifier.height(20.dp))
+
+        Button(
+            onClick = onClickNext,
+            modifier = Modifier
+                .fillMaxWidth(),
+            colors = ButtonDefaults.buttonColors(
+                backgroundColor = colorResource(id = R.color.secondaryColor)
+            )
+        ) {
+            Text(stringResource(R.string.next).uppercase())
+        }
+
+        Spacer(modifier = Modifier.height(20.dp))
+
+        Text(stringResource(R.string.or).uppercase())
+
+        Button(
+            onClick = onClickNewLearningEnvironment,
+            modifier = Modifier
+                .fillMaxWidth(),
+            elevation = null,
+            colors = ButtonDefaults.buttonColors(
+                backgroundColor = Color.Transparent,
+                contentColor = colorResource(id = R.color.primaryColor),
+            )
+        ) {
+
+            Icon(Icons.Filled.Add, contentDescription = "")
+
```

Comment:
Should have a spacer using 

Spacer(Modifier.size(ButtonDefaults.IconSpacing)) 

e.g. as per Google's sample https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/material/material/samples/src/main/java/androidx/compose/material/samples/ButtonSamples.kt